content-src/experiments/send.yaml

```diff
@@ -82,3 +82,13 @@ installation_count: 1
 created: '2017-08-01T14:00:00Z'
 launch_date: '2017-08-01T14:00:00Z'
 order: 2
+news_updates:
+  -
+    slug: send-update-1
```

Comment:
In fact, to keep these things extra unique, maybe stick a date in there - like `send-update-2017-08-10-1`. These slugs need to never be accidentally re-used with new updates later on when old ones have been long since deleted.